test/unit/workDiv/src/WorkDivHelpersTest.cpp

```diff
@@ -34,6 +34,71 @@ namespace
     }
 } // namespace
 
+namespace alpaka::trait
```

Comment:
could you move this to the FooVec, please? No need to have this code here in the test file since it corresponds to the FooVec class.